client/src/components/App.jsx

```diff
@@ -1,25 +1,47 @@
 const styyle = require('../style.scss');
+const axios = require('axios');
+
 import React from 'react';
 
 import Header from './Header.jsx';
 import Subheader from './Subheader.jsx';
 
 class App extends React.Component {
-	constructor(){
-		super();
+	constructor(props){
+		super(props);
+		this.state = {
+			isLoaded: false,
+			details : null
+		}
+	}
+
+	componentDidMount () {
+		axios.get('/details/5')
+			.then(res => {
+				const resturant = res.data;
+				this.setState({ isLoaded: true, details: resturant });
+			})
 	}
 
 	render() {
-		return(
-			<div className="main-container">
-				<div className="board-body">
-					<Header />
+		const { error, isLoaded, details } = this.state;
```

Comment:
Where would error be defined?  Why did you choose to use this isLoaded logic over placing "Loading..." in the initial app component that react binds this to?

Reply:
nevermind!  I had misread your code